iothub/service/tests/ExportImportDeviceTests.cs

```diff
@@ -89,22 +48,22 @@ public void ExportImportDeviceTakingDeviceWithNullEtag()
             // nothing to Act on
 
             // Assert
-            Assert.IsTrue(exportimportDevice.ETag == null, "ETag was not set correctly");
-            Assert.IsTrue(exportimportDevice.TwinETag == null, "Twin ETag was not set correctly");
+            Assert.IsTrue(exportimportDevice.ETag.ToString() == "", "ETag was not set correctly");
```

Comment:
Consider taking the time to convert to fluent assertions
```suggestion
            exportimportDevice.ETag.Should().Be("", "ETag was not set correctly");
```

Reply:
Also, this way it prints out the value if it isn't what was expected.